packages/utils/regl-renderer/types/rendering/commands/drawMesh/meshShaders.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const meshVert: "\nprecision mediump float;\n\nuniform float camNear, camFar;\nuniform mat4 model, view, projection;\n\nattribute vec3 position, normal;\n\nvarying vec3 surfaceNormal, surfacePosition;\nvarying vec4 _worldSpacePosition;\n\nvoid main() {\n  surfacePosition = position;\n  surfaceNormal = normal;\n  vec4 worldSpacePosition = model * vec4(position, 1);\n  _worldSpacePosition = worldSpacePosition;\n\n  vec4 glPosition = projection * view * model * vec4(position, 1);\n  gl_Position = glPosition;\n}\n";
```

Comment:
The redeclaration of these massive string seems wasteful. Are these declarations useful?